### PR TITLE
ADD pytest.ini sample

### PR DESCRIPTION
**Pull request by bealdav**
Provide a template to make easy to use pytest

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi. I don't use docky. I so removed me from the reviewer.
regards !

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*pytest.ini
```
Comment (hparfr) [inline .gitignore:1]: why ?

File: odoo/pytest-sample.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[pytest]
+; Copy this file as pytest.ini in your working directory
+addopts = --odoo-database=dbtest
```
Comment (hparfr) [inline odoo/pytest-sample.ini:3]: use copier instead

File: .gitignore (hunk 1 above)
Comment (bealdav) [reply .gitignore:1]: because it's a personal file with your preferences comes from a template (sample)

**Pull request closed without merging**